basaltinc/bedrock
# React template renderer

🎉  React rendering lands as a Bedrock template renderer!

To test: 

```
yarn
yarn build
cd examples/multi-templates
yarn start
```

Note that the pattern "Card" has a drop down to change between React, Twig php, and HTML.
